xclim/indices/_multivariate.py

```diff
@@ -225,7 +225,7 @@ def daily_freezethaw_cycles(
 
 @declare_units("K", tasmax="[temperature]", tasmin="[temperature]")
 def daily_temperature_range(
-    tasmin: xarray.DataArray, tasmax: xarray.DataArray, freq: str = "YS"
+    tasmin: xarray.DataArray, tasmax: xarray.DataArray, freq: str = "YS", op: str = "mean"
 ) -> xarray.DataArray:
     r"""Mean of daily temperature range.
```

Comment:
```suggestion
    r"""Statistics of daily temperature range.
```

Reply:
The docstring should also be modified I think!

Reply:
@aulemahal @huard @Zeitsperre 
going through the cosmetic chagnes on docstring but saw that in the notes section for `daily_temperature_range`  math is explicitly for original calc i.e. `op=mean`  .... opinions?  leave math with indication that this is 'mean' only or scrap it altogether?

Reply:
I suggest first option.